tests/c/dyn_ptradd_mixed.newcg.c

```diff
@@ -19,10 +19,10 @@
 //     ...
 //     --- End jit-pre-opt ---
 //     i=3, y=6
-//     jitstate: enter-jit-code
+//     yk-jit-event: enter-jit-code
```

Comment:
Dumb question: whats is the definition of a `jit-event` under this new model? I was expecting `enter-jit-code` to be printed differently from state transitions, since entering jit-code doesn't actually cause a state change.

Reply:
Exactly! I was expecting those two things to be formatted differently.

Reply:
`enter-jit-code` is a JIT state change, not a location state change. "is-compiling" (or whatnot) would be a location state change. [FWIW, this PR only partially fixes this sort of thing. A full job requires a complete audit of all of ykrt, because we haven't got the right hooks in the right places yet. It's not a small job, and I don't want to do it right now.]

Reply:
Agreed.

I was expecting classes like:
 - warning (e.g. trace too long)
 - loc-state-change: (e.g. location 0x123 went from counting to tracing)
 - jit-event: (e.g. enter jit code)

But I'm now pretty sure I've expected too much in one go, so I won't persist now.